src/components/network/network-renderer.js

```diff
@@ -4,6 +4,12 @@
 
 'use strict';
 
+/** @private @const {number} */
+genotet.NetworkRenderer.prototype.MAX_TICK_NUM_ = 100;
```

Comment:
This will not work. You need to put this after genotet.NetworkRenderer is defined.